docs/updating.rst

```diff
@@ -1,36 +1,53 @@
-===============================================================
-How to update the Atomic Developer Bundle to the latest version
-===============================================================
+===========================================================
+How to update Atomic Developer Bundle to the latest version
+===========================================================
 
-* If you are using the image from atlas.hashicorp.com
+Depending on your implementation of the ADB, you might be able to directly
+update your Vagrant box. In some cases, you will need to perform a clean
+installation or follow additional steps to complete the update.
 
-  1. Check if a newer version of the box is available use this command::
+-------------
+Prerequisites
+-------------
 
-      # vagrant box outdated
+If you cannot directly update your Vagrant box, you must perform the following steps
+to remove the previously-installed boxes from your virtual machine.
 
-  2. If a newer version is available, the output should look like this::
+1. If ADB is running, run the `vagrant halt` command.
```

Comment:
Also note that "vagrant halt" is not visible in code block. I believe it should be wrapped inside double back ticks (rst format)